Tests/Fixtures/TestProject/carthage_dynamic.xcconfig

```diff
@@ -1 +1,6 @@
 #include "xcode12_and_13_workaround.xcconfig"
+
+IPHONEOS_DEPLOYMENT_TARGET=14.0
+MACOSX_DEPLOYMENT_TARGET=10.15
+TVOS_DEPLOYMENT_TARGET=14.0
+WATCHOS_DEPLOYMENT_TARGET=7.0
```

Comment:
Xcode 14 stopped setting default values for these settings which usually only results in warnings, but in this instance the builds were failing entirely. I came to this solution from https://stackoverflow.com/a/75729977